.github/workflows/haxe-language-server.yml

```
@@ -3,37 +3,39 @@ name: Package haxe-language-server
 on:
   workflow_dispatch:
     inputs:
-      haxe-language-server_version:
+      vshaxe_version:
         type: string
-        required: false
-        default: "1.0.0"
-
+        required: true
 jobs:
   package_all:
     name: Package haxe-language-server
     runs-on: ubuntu-latest
     permissions: write-all
     env:
-      VERSION: ${{ inputs.haxe-language-server_version }}
+      VERSION: ${{ inputs.vshaxe_version }}
     steps:
       - uses: actions/checkout@v4
       - name: Update Packages
         run: sudo apt-get update -y
       - name: Install Dependencies
         run: sudo apt-get install -y npm nodejs git
       - name: Get haxe-language-server
-        run: git clone https://github.com/vshaxe/haxe-language-server
+        run: |
+          git clone --depth 1 --branch ${VERSION} https://github.com/vshaxe/vshaxe.git
+          cd vshaxe
+          git submodule update --init --recursive
+          cd ..
```

You could do something like
```suggestion
          git clone --depth 1 --branch ${VERSION} --recurse-submodules=server https://github.com/vshaxe/vshaxe.git
```

---

Agree